components/Vote/ElectionCandidacy.js

```diff
@@ -284,149 +284,156 @@ class ElectionCandidacy extends React.Component {
       recommendation: candidate ? candidate.recommendation : undefined
     }
 
+    if (new Date() >= election.candidacyEndDate) {
+      return (
+        <Frame meta={meta}>
+          <NarrowContainer>
+            <P>{ vt('vote/candidacy/tooLate') }</P>
+          </NarrowContainer>
+        </Frame>
+      )
+    }
+
     return (
       <Frame meta={meta}>
-        <Loader loading={data.loading} error={data.error} render={() =>
```

Comment:
why is this loader removed? when SSRing and everything works it might look correct, but errors will not show up and when client navigation one might start filling in data that can be pre-filled. Additionally the too could flash in at any point.

Reply:
By accident.